src/tools/pub/cmd/pub-global.md

```diff
@@ -110,8 +108,42 @@ and passes in two arguments.
 $ pub global run foo:bar arg1 arg2
 {% endprettify %}
 
+
 ### Running a script from your PATH
 
+In order to run a script directly from your path, it must be specified
+under the `executables` tag in its pubspec file.
```

Comment:
Except if it isn't listed under `executables` in the pubspec, then adding the bin directory to your path does no good - you can't run it directly, but have to use "pub global run". At least that's what I found when testing this out.